README.md

```diff
@@ -20,21 +60,74 @@ Software Engineering 2 class kaggle competition repo. We'll also be building a b
       ```
 3. Install [SciKit-Learn](http://scikit-learn.org/stable/install.html) using pip
 
-###### Miscellaneous
-- We are using [ZenHub](https://www.zenhub.com/) for project management.
-- We are using [AzureMLStudio](https://studio.azureml.net/) for initial data analysis and some machine learning.
-- Our [Google Drive](https://drive.google.com/drive/folders/0B_C34Fpc9Zf_TDlRUDhlZ0c3WVE?usp=sharing) with our other documents can be viewed here: 
+#### R and Related Libraries
+[Install latest version of R](https://cran.r-project.org/mirrors.html)
+
+#### Javascript, NodeJS, and Related Libraries
+[Follow the instructions here, including submodules](https://github.com/ASAAR/SE2-KaggleComp/blob/master/modules/MLTA/README.md)
+
+#### Test Data
+[Download and setup the test data so that unit tests run properly](https://github.com/ASAAR/SE2-KaggleComp/blob/master/datasets/README.md)
+
+To verify correct setup, please run the tests.
+
+## Running the tests
+
+Go to the top directory of the project and run the following,
+
+```
+python -m unittest discover
+```
+
+This will run tests against every module, including ones that test R and Javascript modules. It will not run all tests, but every module will be covered.
```

Comment:
Yes, because your APIs aren't finished. As soon as your APIs are done it will be false because I'll need to use your APIs and write new tests.